tests/testthat/test-live.R

```diff
@@ -14,11 +14,17 @@ test_that("Scenario Test", {
   cluster <- doAzureParallel::makeCluster(clusterFileName)
   doAzureParallel::registerDoAzureParallel(cluster)
 
+  opt <- list(wait = FALSE)
   '%dopar%' <- foreach::'%dopar%'
-  res <- foreach::foreach(i = 1:4) %dopar% {
+  res <- foreach::foreach(i = 1:4, .options.azure = opt) %dopar% {
```

Comment:
Can we make this a separate test? 

One - Basic scenario test
Two - Basic long running job scenario test

In the long running job scenario test,
1) Wait for job to complete - Sleep
2) Get the job results
3) Check the job results if they are correct
4) Delete the job